package.json

```diff
@@ -29,85 +29,46 @@
     "migrate-create": "yarn workspace @tupaia/database migrate-create",
     "migrate-down": "yarn workspace @tupaia/database migrate-down",
     "package-json-lint": "npmPkgJsonLint --configFile .npmpackagejsonlintrc.json .",
-    "postinstall": "test -n \"$SKIP_BUILD_INTERNAL_DEPENDENCIES\" || yarn build:internal-dependencies",
     "refresh-database": "yarn workspace @tupaia/database refresh-database --root ../../",
     "test-all": "node ./scripts/node/testAll --silent",
-    "validate": "./scripts/bash/validate.sh"
+    "validate": "./scripts/bash/validate.sh",
+    "package:build:ts": "cd $INIT_CWD && tsc -p tsconfig-build.json",
```

Comment:
Scripts with colon can be called from any workspace. Combined with $INIT_CWD which is the workspace calling the script, we can reuse scripts in our workspaces. The other handy thing about this is that yarn loads root level npm packages, so each package doesn't need to have a copy of e.g. eslint declared as a dependency.